lib/shopify_api/resources/base.rb

```diff
@@ -28,6 +28,32 @@ def as_json(options = nil)
     end
 
     class << self
+      def search(params = {})
+        find(:all, params: params)
+      end
```

Comment:
@kevinhughes27 This was a poor name for this: I updated the PR to name it `list`. The `find :all` call doesn't actually search it grabs a list from the shopify endpoints